Description: Tweaks build.properties to work with eclipse-debian-helper
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/org.eclipse.jdt.debug/build.properties
+++ b/org.eclipse.jdt.debug/build.properties
@@ -18,6 +18,7 @@
                META-INF/,\
                .options
 
+source.. = model/,jdi/,eval/
 source.jdimodel.jar = model/,\
                       jdi/,\
                       eval/
